json/src/spec/builtin.rs

```diff
@@ -57,6 +57,41 @@ pub struct AltBn128Pairing {
 	pub pair: u64,
 }
 
+
+/// Bls12 pairing price
+#[derive(Debug, PartialEq, Deserialize, Clone)]
+#[serde(deny_unknown_fields)]
+pub struct Bls12PairingPrice {
```

Comment:
Perhaps rename to `Bls12Pairing` for consistency with the structs immediately above this one (which don't have `Price` as a suffix).